### DISCUSSION TITLE
Change 'byte' to 'u8'

### DISCUSSION THREAD

**Discussion by mattbettcher** [vlang/v#11623] [Ideas]
Can we at least alias this? Just to make the types uniform.

**Comment by crthpl** [comment]
We shouldn't alias it, because of V's 'one way' philosophy.
`byte` makes more sense, as that is what an 8 bit unsigned integer. It is like `int` instead of `i32`. However, the error should tell you to use byte:
```v
test.v:2:7: error: unknown function: u8, did you mean byte?
    1 | fn main() {
    2 |     x := u8(8)
      |          ~~~~~
    3 | }
```
